react/components/CategoryItem.js

```diff
@@ -14,69 +14,77 @@ import categoryMenu from '../categoryMenu.css'
  */
 export default class CategoryItem extends Component {
   state = {
-    isHover: false,
+    isOnHover: false,
   }
 
   static propTypes = {
     /** Category to be displayed */
     category: categoryItemShape.isRequired,
     /** Set use of Link component */
     noRedirect: PropTypes.bool,
-    /** Whether to show subcategories or not */
+    /** Number of subcategory levels */
     subcategoryLevels: PropTypes.oneOf([0, 1, 2]),
   }
 
-  handleCloseMenu = () => (this.setState({ isHover: false }))
+  handleCloseMenu = () => (this.setState({ isOnHover: false }))
 
-  render() {
-    const { category, subcategoryLevels } = this.props
-    const { isHover } = this.state
-
-    const containerStyle = {
-      top: this.item && this.item.offsetTop + this.item.clientHeight,
-      display: isHover ? 'flex' : 'none',
-    }
+  renderCategory(){
+    const { category, noRedirect } = this.props
+    const { isOnHover } = this.state
 
     const linkClasses = classNames(
       'w-100 pv5 mh6 no-underline t-small outline-0 db tc link truncate bb bw1 c-muted-1', {
-        'b--transparent': !isHover,
-        'b--action-primary': isHover,
+        'b--transparent': !isOnHover,
+        'b--action-primary': isOnHover,
       }
     )
 
-    const itemClasses = `${categoryMenu.container} flex justify-center items-center`
+    return noRedirect ? (
+      <a href="#" className={linkClasses}>
```

Comment:
Why we use this approach?

Reply:
seems awkward

Reply:
It's used to render the 'Departments' category. Now its using a span. If we decided to render the departments separately we would have duplicated code for rendering the title and the children.